<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindows</class>
 <widget class="QMainWindow" name="ServerWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="text_chat">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_chat"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="text_console">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOption"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOption">
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
